nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -744,6 +760,13 @@ DockingServer::dynamicParametersCallback(std::vector<rclcpp::Parameter> paramete
       if (name == "max_retries") {
         max_retries_ = parameter.as_int();
       }
+    } else if(type == ParameterType::PARAMETER_BOOL){
+      if (name == "initial_rotation") {
+        initial_rotation_ = parameter.as_bool();
+      }
+      if (name == "dock_backwards_without_sensor") {
```

Comment:
Dock backward isn't a dynamic parameter either, if you're going to enable this one, I think it should be both.